uint32 invocation_id
uint32 pipeline_id
uint32 puser_id
uint32 face_id
time ts
sensor_msgs/Image thumb

